src/config.ts

```diff
@@ -0,0 +1,65 @@
+const hasDOM = typeof document !== 'undefined' && typeof window !== 'undefined';
+const scriptElement: HTMLScriptElement = hasDOM && document.currentScript;
+// @ts-ignore
+const globalConfig: Record<String, string | string[] | null> =
+	globalThis.Prism?.constructor?.name === 'Object' ? globalThis.Prism : {};
+
+function getGlobalSetting (name: string) {
+	let camelCaseName = name.replace(/-([a-z])/g, g => g[1].toUpperCase());
```

Comment:
```suggestion
	let camelCaseName = name.replace(/-([a-z])/g, g => g[1].toUpperCase()); // eslint-disable-line regexp/no-unused-capturing-group
```

ESLint still can't analyze the code properly. 😅